### ISSUE TITLE
Installation Kimai 2 stops when installing ocramius/package-versions 1.4.2

### ISSUE THREAD

**Issue by green-bits** [kevinpapst/kimai2#1410]
I am trying to install Kimai 2 1.7 following the instructions on 
https://www.kimai.org/documentation/installation.html#recommended-setup
but the package ocramius 1.4.2 is not installed.
The installation stops when running the command 'composer install --no-dev --optimize-autoloader' with the following  warnings / error:
```

PHP Warning:  Unsupported declare 'strict_types' in /xxx/kimai/kimai2/vendor/ocramius/package-versions/src/PackageVersions/Installer.php on line 3

Warning: Unsupported declare 'strict_types' in /xxx/kimai/kimai2/vendor/ocramius/package-versions/src/PackageVersions/Installer.php on line 3
PHP Parse error:  syntax error, unexpected ':', expecting ';' or '{' in /xx/kimai/kimai2/vendor/ocramius/package-versions/src/PackageVersions/Installer.php on line 77

Parse error: syntax error, unexpected ':', expecting ';' or '{' in /xxx/kimai/kimai2/vendor/ocramius/package-versions/src/PackageVersions/Installer.php on line 77
```

My PHP version is 7.1.33.
What can I do? Can I skip the installation of ocramius?

**Comment by kevinpapst** [comment]
PHP 7.1 is not supported, see https://www.kimai.org/download/
You need at least 7.2.x

**Comment by green-bits** [comment]
I also tried the installation with PHP 7.2., at least phpinfo() showed me this version. But now I found the error. I tried the installation on an all-inkl.com account. Even though I set the correct php version for my subdomain, internally version 5.6 was used. 
To internally use the correct version I needed to prefix /usr/bin/php72
So for me 
`/usr/bin/php72 /www/htdocs/xxx/bin/composer.phar install --no-dev --optimize-autoloader`
worked.
